orbotservice/src/main/java/org/torproject/android/service/OrbotService.java

```diff
@@ -425,6 +428,7 @@ protected void logNotice(String msg) {
         }
     }
 
+    @SuppressWarnings("ResultOfMethodCallIgnored")
```

Comment:
This seems like a mistake, `onCreate()` is void and I don't see this warning on my machine

Reply:
This is for code *inside* #onCreate. Esp. #mkdirs.